sources/lib/reasoners/sat_solver.ml

```diff
@@ -18,70 +18,4 @@ let get_current () =
   | Util.CDCL | Util.CDCL_Tableaux ->
     if Options.verbose() then
       Format.eprintf "[bool reasoning] use CDCL solver@.";
-    (module Satml_frontend : Sat_solver_sig.SatContainer)
-
-(*
-(*+ no dynamic loading of SAT solvers anymore +*)
-
-open Options
-open Format
-
-let current = ref (module Fun_sat : Sat_solver_sig.S)
-
-let initialized = ref false
-
-let set_current sat =
-  if use_satml() then
-    current := sat
-
-let load_current_sat () =
-  match sat_plugin () with
-  | "" ->
-    if debug_sat () then eprintf "[Dynlink] Using Fun_sat solver@."
-
-  | path ->
-    if debug_sat () then
-      eprintf "[Dynlink] Loading the SAT-solver in %s ...@." path;
-    try
-      MyDynlink.loadfile path;
-      if debug_sat () then  eprintf "Success !@.@."
-    with
-    | MyDynlink.Error m1 ->
-      if debug_sat() then begin
-        eprintf
-          "[Dynlink] Loading the SAT-solver in plugin \"%s\" failed!@."
-          path;
-        Format.eprintf ">> Failure message: %s@.@."
-                             (MyDynlink.error_message m1);
-      end;
-      let prefixed_path = sprintf "%s/%s" Config.pluginsdir path in
-      if debug_sat () then
-        eprintf "[Dynlink] Loading the SAT-solver in %s ... with prefix %s@."
-          path Config.pluginsdir;
-      try
-        MyDynlink.loadfile prefixed_path;
-        if debug_sat () then  eprintf "Success !@.@."
-      with
-      | MyDynlink.Error m2 ->
-        if not (debug_sat()) then begin
-          eprintf
-            "[Dynlink] Loading the SAT-solver in plugin \"%s\" failed!@."
-            path;
-          Format.eprintf ">> Failure message: %s@.@."
-            (MyDynlink.error_message m1);
-        end;
-        eprintf
-          "[Dynlink] Trying to load the plugin from \"%s\" failed too!@."
-          prefixed_path;
-        Format.eprintf ">> Failure message: %s@.@."
-                             (MyDynlink.error_message m2);
-        exit 1
-
-let get_current () =
-  if not !initialized then
-    begin
-      load_current_sat ();
-      initialized := true;
-    end;
-  !current
- *)
+    (module Satml_frontend : Sat_solver_sig.SatContainer)
```

Comment:
```suggestion
    (module Satml_frontend : Sat_solver_sig.SatContainer)

```

Reply:
Git prefers when there's a new line at end of file and I won't argue with git on this matter